js/components/select.tsx

```diff
@@ -35,6 +35,7 @@ class Select extends React.Component<IItemsComponentProps, IDropDownState> {
         this.isScrollLimit = this.isScrollLimit.bind(this);
 
         this.debouncedOnSearch = manywho.utils.debounce(this.props.onSearch, 750);
+        this.debouncedOnScroll = manywho.utils.debounce(this.isScrollLimit, 100);
```

Comment:
Not 100% sure, but I suspect `debouncedOnScroll` might need a class member like `debouncedOnSearch` has (at the top of the class, line 22) as TypeScript is complaining about it.

Reply:
Added     debouncedOnScroll = null;